packages/bitcore-wallet-service/src/lib/server.ts

```diff
@@ -1019,7 +1032,8 @@ export class WalletService {
     if (_.isEmpty(opts.name)) return cb(new ClientError('Invalid copayer name'));
 
     opts.coin = opts.coin || Defaults.COIN;
-    if (!Utils.checkValueInCollection(opts.coin, Constants.COINS)) return cb(new ClientError('Invalid coin'));
+    // checking in chains for simplicity
+    if (!Utils.checkValueInCollection(opts.coin, Constants.CHAINS)) return cb(new ClientError('Invalid coin'));
```

Comment:
Added. I wasn't sure about changing createWallet and joinWallet since chain === coin when we are storing clients. But I decided to refactor both and I forgot about joinWallet